ZelBack/src/services/fluxCommunication.js

```diff
@@ -40,13 +40,13 @@ const LRUoptionsTemp = { // cache for temporary messages
 
 const myCacheTemp = new LRUCache(LRUoptionsTemp);
 
-/* const LRUTest = {
+const LRUTest = {
```

Comment:
this is just for dev, it will be comment out again, as I want to see how many messages the dev nodes are currently receiving every hour as we had a rise on apps on the network.

Reply:
block already commented again.